Api/DTO/Posts/PostDto.cs

```diff
@@ -9,4 +10,6 @@ public record PostDto(
     string EventTitle,
     PublicUserDto? Author,
     DateTime CreationDate,
+    ReactionType? UserReaction,
+    int ReactionCount,
```

Comment:
I think it should be better to show how many reactions of each type are there, just an opinion as I don't know how it is shown on frontend.

Reply:
I added a summary to instead